bin/index.js

```diff
@@ -35,13 +36,19 @@ var getNetworkAddress = () => {
         .option("p", { alias: 'port', describe: "Change default port", type: "integer", demandOption: false })
         .option("ip", { describe: "Your machine public ip address", type: "string", demandOption: false })
         .option("c", { alias: 'clipboard', describe: "Share Clipboard", type: "boolean", demandOption: false })
+        .option("w", { alias: 'windows', describe: "Is windows os", type: "boolean", demandOption: false })
         .help(true)
         .argv;
 
 
     let path = undefined;
     let fileName = undefined;
 
+    if (options.windows) {
```

Comment:
Hi @Seven-Y-Q-Guo 
another nice PR ^_^
I think we can remove the `-w` option and change this to something like this:
```
if (process.platform === 'win32') {
   ... Do stuff for windows
}
```
Thanks a lot

[process.platform](http://nodejs.org/api/process.html#process_process_platform)

Reply:
Yeal, I was also thought about this idea in my mind, but after confirming the issue will only happen on Windows via native terminal. If use others like powershell, git bash, etc. It'll be working fine.
Use this logic will make qrcode big than ever before, does it make sense?

Reply:
you're right. in that case, we need to make this option name and description a little more accurate.
```
.option("on-windows-native-terminal", { describe: "enable QR-Code support for windows native terminal", type: "boolean", demandOption: false })
```
the above line is just an example, you may come up with a better name and description. 

great regards